storage_v2/factoryadapter/reader.go

```diff
@@ -62,21 +63,28 @@ func (tr *TraceReader) GetOperations(ctx context.Context, query tracestore.Opera
 	return operations, nil
 }
 
-func (*TraceReader) FindTraces(_ context.Context, _ tracestore.TraceQueryParameters) ([]ptrace.Traces, error) {
-	panic("not implemented")
+func (tr *TraceReader) FindTraces(
+	ctx context.Context,
+	query tracestore.TraceQueryParameters,
+) ([]ptrace.Traces, error) {
+	t, err := tr.spanReader.FindTraces(ctx, query.ToSpanStoreQueryParameters())
+	if err != nil || t == nil {
+		return nil, err
+	}
+	otelTraces := []ptrace.Traces{}
+	for _, trace := range t {
+		batch := &model.Batch{Spans: trace.GetSpans()}
+		otelTrace, err := model2otel.ProtoToTraces([]*model.Batch{batch})
+		if err != nil {
+			return nil, err
+		}
```

Comment:
@yurishkuro I can't test this branch because `ProtoToTraces` never actually returns a non-nil error (https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/translator/jaeger/jaegerproto_to_traces.go#L27)

Reply:
maybe ignore the error then, I think I did the same in api_v3 handler